x-pack/plugins/fleet/common/types/index.ts

```diff
@@ -61,6 +61,10 @@ export interface FleetConfigType {
     };
   };
   createArtifactsBulkBatchSize?: number;
+  packageUpgrade?: {
```

Comment:
I am wondering if it make more sense to have these flags on the install package endpoint, so we can apply this on case per case basis, if an error happen or a package upgrade is stuck, wdyt?

Reply:
Yeah that could be easier to apply than a kibana config, the only caveat is that we can't stop Fleet setup to re-run the upgrade if a preconfigured package has an issue. 
Though if the API call succeeds, the setup should not try to reinstall the package, so it might be enough to have the flags on the API.